offline/docs.md

```diff
@@ -324,26 +324,30 @@ For full docs with details and explanations please see https://github.com/wireap
 First, make sure you have a certificate for `sftd.<yourdomain>`. This could be the same wildcard or SAN certificate
 you used at previous steps.
 
-If you want to restrict SFT to certain nodes, make sure that in your inventory
-you have annotated all the nodes that are able to run sftd workloads correctly.
+Next, copy the values/sftd/prod-example-values.yaml to values/sftd/values.yaml, and change the contents accordingly. 
+
+ * If your turn servers can be reached on their public IP by the SFT service, Wire recommends you enable cooperation between TURN and SFT. add a line reading `turnDiscoveryEnabled: true` to your values file.
```

Comment:
Is it a `turn server` or a `TURN server`? (If `turn` is a name, we should write and format it everywhere the same...)